packages/fast-usdc/src/cli/operator-commands.js

```diff
@@ -1,18 +1,42 @@
-/* eslint-env node */
 /**
  * @import {Command} from 'commander';
  * @import {OfferSpec} from '@agoric/smart-wallet/src/offers.js';
  * @import {ExecuteOfferAction} from '@agoric/smart-wallet/src/smartWallet.js';
+ * @import {OperatorKit} from '../exos/operator-kit.js';
  */
 
 import { fetchEnvNetworkConfig, makeVstorageKit } from '@agoric/client-utils';
+import { mustMatch } from '@agoric/internal';
+import { Nat } from '@endo/nat';
+import { InvalidArgumentError } from 'commander';
 import { INVITATION_MAKERS_DESC } from '../exos/transaction-feed.js';
+import { CctpTxEvidenceShape } from '../type-guards.js';
 import { outputActionAndHint } from './bridge-action.js';
 
+const parseNat = arg => {
+  const n = Nat(BigInt(arg));
```

Comment:
Is this because `arg` is a string? please annotate.

If it's a Number the following suffices,
```suggestion
  const n = Nat(arg);
```

Reply:
yes, [custom option callbacks](https://www.npmjs.com/package/commander#custom-option-processing) are called with a string. annotated.